### PR TITLE
double-click middle button

### PR DESCRIPTION
**Pull request by amirfru**
Added an option to double-click the middle button to bypass the addon's function

### CODE REVIEW

**Comment by StoyanDimitrov** [comment]
Thank you for your contribution. Unfortunately, I'm currently away. Will handle this PR in the first weeks of the next month.

File: content_scripts/scroll.js [hunk 1]
```diff
@@ -92,6 +92,9 @@
     }
   }
 
+  let doubleClick = false
+  let doubleClick_time = 500
```
Comment (StoyanDimitrov) [inline content_scripts/scroll.js]: Stick to the camel case notation.

File: content_scripts/scroll.js [hunk 2]
```diff
@@ -113,6 +116,15 @@
       return
     }
 
+    if (doubleClick) {
```
Comment (StoyanDimitrov) [inline content_scripts/scroll.js]: Check for the real value here not a truthy one

File: content_scripts/scroll.js [hunk 3]
```diff
@@ -113,6 +116,15 @@
...
+        return
+    }
+    else
+    {
```
Comment (StoyanDimitrov) [inline content_scripts/scroll.js]: Else case is unneeded here.

File: content_scripts/scroll.js [hunk 4]
```diff
@@ -113,6 +116,15 @@
...
+        doubleClick = true
+        setTimeout(function(){ doubleClick = false }, doubleClick_time);
```
Comment (StoyanDimitrov) [inline content_scripts/scroll.js]: Use arrow function as the rest of the code

**Review by StoyanDimitrov — CHANGES_REQUESTED** [review]

**Comment by StoyanDimitrov** [comment]
Thank you for your patience.

**Comment by amirfru** [comment]
No prob,
I've made the changes :)

**Comment by StoyanDimitrov** [comment]
Thank you!

**Comment by amirfru** [comment]
Sure, nice addon, cheers!

**Pull request merged**